src/builder/command.rs

```diff
@@ -708,6 +708,52 @@ impl<'help> App<'help> {
         c.print()
     }
 
+    /// Returns the help message (`-h`).
```

Comment:
Please move these functions to a consistent location (we currently have short/long help grouped and show `print` followed by `write` variants).